magda-authorization-api/src/index.ts

```diff
@@ -39,6 +39,11 @@ const argv = addJwtSecretFromEnvVar(
         .option("jwtSecret", {
             describe: "The shared secret for intra-network communication",
             type: "string"
+        })
+        .option("tenantId", {
```

Comment:
No. It does not make sense. However, this PR does not address authorization in multi-tenant mode. This is a temporary solution only. In the future, the api will add a directive that demands all requests have tenant id header.

Reply:
Yep ok auth in multi-tenant mode can be a future thing. There are two (future) steps:
1. Make the auth API services like `/public/users/allowed/dataset` extract the `tenantId` from the request header, instead of using a `tenantId` specified on the command line. This should be pretty straightforward and once it's implemented auth will work in multi-tenant mode, but users will still be global rather than per-tenant.
2. Add true multi-tenancy to the auth API. This will be a bigger change and will mean the users are scoped to a particular tenant just like registry records are.

Reply:
done